### ISSUE TITLE
Support @EntiryScan

### ISSUE THREAD

**Issue by varkula** [arangodb/spring-boot-starter#16]
Is it possible to support @EntiryScan in the autoconfiguration?

I.e. override ArangoMappingContext arangoMappingContext
and use EntityScanner in creation of MappingContext like mappingContext.setInitialEntitySet(new EntityScanner(applicationContext).scan(Document.class, Edge.class, TypeAlias.class));

**Comment by rashtao** [comment]
Thanks for reporting. As of now you can manually configure it, i.e. see https://github.com/arangodb/spring-data/issues/224#issuecomment-933369080

**Comment by varkula** [comment]
Yes, this is what I did, but in this case I have to implement ArangoConfiguration which in turn disables ArangoAutoconfiguration
